dockstore-launcher/mock.txt

```diff
@@ -0,0 +1,4 @@
+one
```

Comment:
Presumably these two text files are testing resources, can they be moved to src/main/resources?